packages/api/src/models/team.ts

```diff
@@ -39,6 +41,11 @@ export default mongoose.model<ITeam>(
         type: Boolean,
         default: false,
       },
+      // TODO: maybe add these to a top level Mixed type
+      // CH Client Settings
+      maxRowsToRead: Number,
```

Comment:
I don't think this naming makes sense, as we only use this for metadata scanning.

Reply:
My original intent was to make this more generic, but I guess it will probably only be used for metadata going forward. I'll go ahead and rename it